pkg/storkctl/migrationschedule.go

```diff
@@ -43,22 +52,79 @@ func newCreateMigrationScheduleCommand(cmdFactory Factory, ioStreams genericclio
 				util.CheckErr(fmt.Errorf("ClusterPair name needs to be provided for migration schedule"))
 				return
 			}
+
+			//Identify whether it is a Sync-Dr or Async-Dr use-case from the ClusterPair spec
+			migrationScheduleNs := cmdFactory.GetNamespace()
+			clusterPairObj, err := storkops.Instance().GetClusterPair(clusterPair, migrationScheduleNs)
+			if err != nil {
+				util.CheckErr(fmt.Errorf("unable to find the cluster pair in the given namespace"))
+				return
+			}
+			isSyncDr := false
+			//clusterPairObj.Spec.Options map is empty for syncDr and non-empty for asyncDR
+			if len(clusterPairObj.Spec.Options) == 0 {
+				isSyncDr = true
+			}
+
+			// Default value of includeVolumes for asyncDr is true and for sync DR is false
+			if isSyncDr {
+				//covers the unlikely scenario where user sets the --exclude-volumes=false value in the command for a sync-dr migration use case
+				if c.Flags().Changed("exclude-volumes") && includeVolumes {
+					util.CheckErr(fmt.Errorf("--exclude-volumes can only be set to true in case of a sync-dr usecase as there is a single stretched cluster from storage perspective"))
+					return
+				}
+				includeVolumes = false
+			}
+
 			if len(namespaceList) == 0 {
 				util.CheckErr(fmt.Errorf("need to provide atleast one namespace to migrate"))
 				return
 			}
-			if len(schedulePolicyName) == 0 {
-				util.CheckErr(fmt.Errorf("need to provide schedulePolicyName"))
+
+			//There are 4 possible cases for schedulePolicyName and interval flags
+			//user provides both schedulePolicyName and interval value -> we throw an error saying you can only fill one of the values
```

Comment:
Number these scenarios starting with 1 as these are getting referred in later comments.

Reply:
Addressed in the revision